nasa-gibs/worldview
# WV-2317: Add Ability To manually input image download extents

## Description
Add Ability To manually input image download extents

Fixes #wv-2317
Fixes #wv-2316

- [x] update react-image-crop version
- [x] make image download container state based on GEO lat-long values instead of pixel boundaries
- [x] Add input component
- [x] Hide input selection by default

[If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...]

## How To Test

1. Open Image download dialog
2. expand the coordinates selection
3. Verify expected result
4. Verify works in polar projections


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview
